src/include/packet_types.h

```diff
@@ -153,6 +153,14 @@ typedef struct altitude_data_block {
     uint8_t bytes[16];
 } TIGHTLY_PACKED AltitudeDataBlock;
 
-void altitude_data_block_init(AltitudeDataBlock *b, const int8_t measurment_type, const uint8_t pressure,
-                              const uint8_t temperature, const uint8_t altitude);
-#endif // _PACKET_TYPES_H
+void altitude_data_block_init(AltitudeDataBlock *b, const uint32_t measurement_time, const int32_t pressure,
```

Comment:
Mismatch between uint32_t and int32_t. Prototype declares int32_t for pressure, however function declares uint32_t.

Reply:
Fixed!